docs/core-concepts/security/security-model.md

```diff
@@ -0,0 +1,200 @@
+
+| **Term** | **Symbol** | **Description** |
+| --- | --- | --- |
+| Max Validator Count |  $n$ | The maximum number of validator nodes participating in the system (currently $n =200$) |
+| Validator Set | $N$ | Set of all the validators. $|N|$ is the total number of validator nodes participating in the system. |
+| Total Chunks | $m$ | The total number of chunks after encoding (currently $m=8192$) |
+| Coding Rate | $r$ | The total number of chunks after encoding / total number of chunks before encoding (currently $r=8$) |
+| Percentage of blob per validator  | $\alpha_i$ | $\alpha_i = rm_i/m$, the percentage of chunks required to reconstruct the blob assigned to validator $i$ |
+| Num of Chunks Assigned | $m_i$ | The number of chunks assigned to validator $i$ |
+| Validator Stake | $\eta_i$  | The stake proportion of validator $i$ ($0 \le \eta_i \le 1$, and $\sum_{i} \eta_i = 1$) |
+| Reconstruction Threshold | $\gamma$  | The minimum percentage of total stake required for a group of validators to successfully reconstruct the blob |
+| Confirmation Threshold | $\eta_C$ | The minimum percentage of stake to make a DA certificate valid |
+| Safety Threshold | $\eta_S$ | The minimum percentage of stake to cause safety failure, $\eta_S = \eta_C - \gamma$ |
+| Liveness Threshold | $\eta_L$ | The minimum percentage of stake to cause liveness failure, $\eta_L = 1 - \eta_C$ |
+
+**Properties**
+
+The assignment logic must satisfy the following properties: 
+
+1. Exact number of chunks: $\sum_i m_i = m$. 
+2. Reconstruction: If a blob passes `VerifySecurityParameters`, then for any set of validators $H \subseteq N$ such that $\sum_{i \in H} \eta_i \ge \gamma$, we must have $\sum_{i\in H} \alpha_i \ge 1$. 
+
+**Specification**
+
+- **GetChunkAssignments**
+
+Let
+
+$$
+m'_i= \left\lceil\eta_i(m- |N|)\right\rceil
+$$
+
+where $|N|$ is the actual number of operators. Note that $\sum_{i} m'_{i} \le   \sum_{i} [\eta_{i} (m-|N|)+1] = m-|N| + |N| = m$. Sort the validator nodes (in a deterministic order), and let $k_i$ be the index of node $i$ within this list.  Let  $m' = \sum_i m’_i$. Finally, let
+
+$$
+m_i=m'_i + \mathbb{I}_{k_i \le m-m'}
+$$
+
+- **VerifySecurityParameters**
+
+Verification succeeds as long as the following condition holds
+
+$$
+n \le m(1 - 1/r\gamma)
+$$
+
+Note that from the inequality above, we can derive that $\gamma \ge \frac{m}{(m-n)r} > 1/r$, which implies that the reconstruction threshold is greater than the theoretical lower bound of stake needed for reconstruction $(1/r)$, due to the chunk assignment logic. 
+
+**Analysis**
+
+We want to show that for a blob which has been distributed using `GetChunkAssignments` and which satisfies `VerifySecurityParameters` , the following properties hold: 
+
+1. Number of allocated chunks is equal to $m$. 
+
+$$
+\sum_i m_i = \sum_i (m'_i + \mathbb{I}_{k_i \le m - m'}) = m ' + \sum \mathbb{I}_{k_i \le m - m'} = m' + m-m' = m
+$$
```

Comment:
This too does not appear to parse right.

![image](https://github.com/user-attachments/assets/70eb759f-e872-497f-926d-c6df32ad2243)